test-case/check-sof-logger.sh

```diff
@@ -71,7 +71,10 @@ sof_alsa_card_found()
     # note: assumes SOF card names to start with "sof", e.g.
```

Comment:
Change is good, but the commit message seems wrong. There are two signed offs. Can you amend the message and repush?